extensions/git/src/commands.ts

```diff
@@ -1236,17 +1236,29 @@ export class CommandCenter {
 		await repository.ignore(uris);
 	}
 
+	@command('git.stashIncludeUntracked', { repository: true })
+	async stashIncludeUntracked(repository: Repository): Promise<void> {
+		if (repository.workingTreeGroup.resourceStates.length === 0) {
+			window.showInformationMessage(localize('no changes stash', "There are no changes to stash."));
+			return;
+		}
+
+		const message = await this.getStashMessage();
+
+		if (typeof message === 'undefined') {
```

Comment:
Are you sure that you need here `typeof`?
You can equal this like: `message === undefined` ok do something.